index.html

```diff
@@ -0,0 +1,30 @@
+<!Doctype <!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Andrew Lyon</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <ul class="nav">
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About Me</a></li>
+        <li><a href="#">Services</a></li>
+        <li><a href="#">Work</a></li>
+        <li><a href="#">Contact</a></li>
+    </ul>
+
+    <div class="container">
+        <div class="row">
+            <div class="col s4"></div>
+    <img class="Image" src="ME.jpeg" alt="Picture of Me">

```

Comment:
good job getting the file path for the image! Next time, let's try putting the image in an image folder, and the image folder inside an assets folder. for example:  assets > images > ME.jpeg , this is a best practice.